packages/yoroi-extension/chrome/extension/background.js

```diff
@@ -1614,6 +1658,28 @@ function handleInjectorConnect(port) {
   });
 }
 
+function metadataIsValid(policyId, assetName, metadatas): any {
+  let metadataWrapper = null;
+  for (const m of metadatas) {
+    if (m.key === '721') {
+      metadataWrapper = m;
+    }
+  }
```

Comment:
All of this can be replaced by single line `metadatas.find(m => m.key === '721');`